### ISSUE TITLE
Can't import [python3]

### ISSUE THREAD

**Issue by maniexx** [kennethreitz/records#3]
I use ubuntu 14.04 LTS. I installed the library system-wide, using `sudo pip3 install records`, it installed successfully. But when I try to import it (in REPL), I get the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/records.py", line 6, in <module>
    import tablib
  File "/usr/local/lib/python3.4/dist-packages/tablib/__init__.py", line 3, in <module>
    from tablib.core import (
  File "/usr/local/lib/python3.4/dist-packages/tablib/core.py", line 15, in <module>
    from tablib import formats
  File "/usr/local/lib/python3.4/dist-packages/tablib/formats/__init__.py", line 6, in <module>
    from . import _csv as csv
  File "/usr/local/lib/python3.4/dist-packages/tablib/formats/_csv.py", line 6, in <module>
    from tablib.compat import is_py3, csv, StringIO
  File "/usr/local/lib/python3.4/dist-packages/tablib/compat.py", line 25, in <module>
    import tablib.packages.xlwt3 as xlwt
ImportError: No module named 'tablib.packages.xlwt3'
```

I checked, and `tablib` is installed. I installed `xlwt3` manually, but that didn't solve (or change) the problem.

**Comment by kennethreitz** [comment]
@maniexx what version of tablib do you have installed?

**Comment by maniexx** [comment]
Version: 0.11.0

**Comment by kennethreitz** [comment]
I will investigate. Downgrading to an earlier version may solve the problem (unfortunately).

**Comment by maniexx** [comment]
@kennethreitz Looks like I can't even import tablib, so it's not records fault. I will try downgrading, and edit this comment if that helps.

**Comment by behrtam** [comment]
tablib 0.10.0 works for me (Python 3.5.1 @ OS X)

**Comment by kennethreitz** [comment]
@maniexx @behrtam I just released tablib v0.11.1, which fixes the packaging error. Sorry about that (grr, contributors!).
